playbooks/libvirt/openshift-cluster/templates/user-data

```diff
@@ -21,3 +21,4 @@ ssh_authorized_keys:
 
 runcmd:
   - NETWORK_CONFIG=/etc/sysconfig/network-scripts/ifcfg-eth0; if ! grep DHCP_HOSTNAME ${NETWORK_CONFIG}; then echo 'DHCP_HOSTNAME="{{ item[0] }}.example.com"' >> ${NETWORK_CONFIG}; fi; pkill -9 dhclient; service network restart
+  - echo "Defaults !requiretty" >> /etc/sudoers.d/00-no-requiretty
```

Comment:
I'd feel a little more comfortable if this were disabling requiretty only for the openshift user: `Defaults:openshift !requiretty`, since this is something that could more generically be used for the other providers that support cloud-init as well.

I'd also suggest using the write_files cloud-init module instead of using runcmd and echo:
```
write_files:
  - path: /etc/sudoers.d/00-openshift-no-requiretty
    permissions: 440
    content: |
        Defaults:openshift !requiretty
```